config.js

```diff
@@ -1,12 +1,7 @@
-const jscodeshiftArgs = [
-	'--extensions=js,jsx',
-];
+const jscodeshiftArgs = [ '--extensions=js,jsx' ];
```

Comment:
Do we need to handle `ts,tsx` as well now?

Reply:
Possibly, although I'm not sure if the tools support that, nor whether there would be any code changes needed.

In any case, there don't appear to be any reducers in `.ts` files at the moment.